### PR TITLE
add BWF embedding recipe

### PR DESCRIPTION
**Pull request by privatezero**
Basic recipe for file writing with BWF Metadata - not exactly a typical 'rewrap' but that felt the closest category?

Ping at all the NTTW jet-setters this week! ;-P

### CODE REVIEW

**Review by retokromer — APPROVED** [review]
LGTM

**Comment by retokromer** [comment]
I think re-wrapping is fine. If I remember carefully, Kate Murray presented it as such.

File: index.html [hunk 1]
```diff
@@ -219,6 +219,36 @@ <h4>Important caveat</h4>
     </div>
     <!-- End Basic rewrap command -->
 
+    <!-- BWF -->
+    <label class="recipe" for="bwf">Convert to (or create) Broadcast WAV</label>
+    <input type="checkbox" id="bwf">
+    <div class="hiding">
+      <h5>Generate Broadcast WAV</h5>
+      <p><code>ffmpeg -i <em>input_file.wav</em> -c copy -write_bext 1 -metadata field_name='Content' <em>output_file.wav</em></code></p>
+      <p>This command will create a write a file in the Broadcast WAV (BWF) containing a BEXT chunk with related metadata.</p>
```
Comment (kieranjol) [inline index.html]: I think this sentence needs to be rewritten with the word format added and "a write" removed?

**Review by kieranjol — COMMENTED** [review]
Great addition,I left one comment about wording but everything else looks great to me.

File: index.html (hunk 1 above)
Comment (privatezero) [reply index.html]: thanks for catching that @kieranjol - I changed my mind on that sentence like three times and clearly subconsciously split the difference!

**Review by kieranjol — APPROVED** [review]

**Comment by ablwr** [comment]
@kieranjol is the best person in the world! Andrew you are also cool

File: index.html [hunk 2]
```diff
@@ -225,7 +225,7 @@ <h4>Important caveat</h4>
     <div class="hiding">
       <h5>Generate Broadcast WAV</h5>
       <p><code>ffmpeg -i <em>input_file.wav</em> -c copy -write_bext 1 -metadata field_name='Content' <em>output_file.wav</em></code></p>
-      <p>This command will create a write a file in the Broadcast WAV (BWF) containing a BEXT chunk with related metadata.</p>
+      <p>This command will write a file in the Broadcast WAV format (BWF) containing a BEXT chunk with related metadata.</p>
```
Comment (retokromer) [inline index.html:228]: Is the official name not Broadcast Wave Format?

**Comment by ablwr** [comment]
oh sorry @retokromer -- feel free to send in a patch to update the formal name!

File: index.html (hunk 2 above)
Comment (dericed) [reply index.html:228]: The official name of `Broadcast Wave Format`, `BWF,` and `Broadcast WAV format` is `Broadcast Wave Format`, see the [EBU spec](https://tech.ebu.ch/docs/tech/tech3285.pdf). 

Happy to see BWF MetaEdit in here :)

**Comment by retokromer** [comment]
@ablwr done in https://github.com/amiaopensource/ffmprovisr/pull/412

**Pull request merged**